config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml

```diff
@@ -37,5 +38,6 @@ spec:
     className: org.apache.flink.streaming.examples.wordcount.WordCount
     args: ["--input", "./README.txt"]
     parallelism: 2
+    restartPolicy: Never
```

Comment:
same here, it is also a required field, if not provided. The admission controller denied create ops

Reply:
This field is optional, the default is "Never", see https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/8fb6191ac0f6728e1b0e89aba49b13a7217f5996/api/v1beta1/flinkcluster_default.go#L116

Reply:
run kubectl get crd flinkclusters.flinkoperator.k8s.io  -oyaml to check if it is required

Reply:
whatever, i will first revet this change. Fix it in a separate pr if needed

Reply:
kubectl apply -f config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml 
error: error validating "config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml": error validating data: [ValidationError(FlinkCluster.spec.job): missing required field "restartPolicy" in io.k8s.flinkoperator.v1beta1.FlinkCluster.spec.job, ValidationError(FlinkCluster.spec.jobManager): missing required field "accessScope" in io.k8s.flinkoperator.v1beta1.FlinkCluster.spec.jobManager]; if you choose to ignore these errors, turn validation off with --validate=false

Reply:
This is validated by kubectl, i think you are creating flinkcluster with client-go directly

Reply:
And the default value is set by admission, so kube-apiserver will not treat is as invalid